src/orion/core/io/resolve_config.py

```diff
@@ -231,9 +233,26 @@ def merge_configs(*configs):
     return merged_config
 
 
+def fetch_user_repo(user_script):
+    """Fetch the GIT repo and its root path given user's script."""
+    dir_path = os.path.dirname(os.path.abspath(user_script))
+    git_repo = git.Repo(dir_path, search_parent_directories=True)
+    git_root = git_repo.git.rev_parse("--show-toplevel")
+    return git_repo, git_root
```

Comment:
Could we drop git_root since it is not used anywhere?

Reply:
Sure.